src/app/page/side-nav/side-nav.component.html

```diff
@@ -16,6 +16,7 @@
       <li><a routerLink="showcase/button" [routerLinkActive]="'is-selected'" class="side-nav__item">Button</a></li>
       <li><a routerLink="showcase/card" [routerLinkActive]="'is-selected'" class="side-nav__item">Card</a></li>
       <li><a routerLink="showcase/list" [routerLinkActive]="'is-selected'" class="side-nav__item">List</a></li>
+      <li><a routerLink="showcase/list-load-more" [routerLinkActive]="'is-selected'" class="side-nav__item">List - load more</a></li>
```

Comment:
```suggestion
      <li><a routerLink="showcase/list/load-on-demand" [routerLinkActive]="'is-selected'" class="side-nav__item">List - Load On Demand</a></li>
```

Reply:
There are problems with child paths and the current menu. I don't want to dive into changes that in this PR. So for now i will leave as a separate path.